localstack/services/ec2/provider.py

```diff
@@ -0,0 +1,206 @@
+                    AvailabilityZone="eu-central-1a",
+                    Duration=2628000,
+                    FixedPrice=0.0,
+                    InstanceType=InstanceType.t2_small,
+                    ProductDescription=RIProductDescription.Linux_UNIX,
+                    ReservedInstancesOfferingId=long_uid(),
+                    UsagePrice=0.0,
+                    CurrencyCode=CurrencyCodeValues.USD,
+                    InstanceTenancy=Tenancy.default,
+                    Marketplace=True,
+                    PricingDetails=[PricingDetail(Price=0.0, Count=3)],
+                    RecurringCharges=[
+                        RecurringCharge(Amount=0.25, Frequency=RecurringChargeFrequency.Hourly)
+                    ],
+                    Scope=scope.Availability_Zone,
+                )
+            ]
+        )
+
+    @handler("DescribeReservedInstances", expand=False)
+    def describe_reserved_instances(
+        self,
+        context: RequestContext,
+        describe_reserved_instances_request: DescribeReservedInstancesRequest,
+    ) -> DescribeReservedInstancesResult:
+        return DescribeReservedInstancesResult(
+            ReservedInstances=[
+                ReservedInstances(
+                    AvailabilityZone="eu-central-1a",
+                    Duration=2628000,
+                    End=datetime(2016, 6, 30, tzinfo=timezone.utc),
+                    FixedPrice=0.0,
+                    InstanceCount=2,
+                    InstanceType=InstanceType.t2_small,
+                    ProductDescription=RIProductDescription.Linux_UNIX,
+                    ReservedInstancesId=long_uid(),
+                    Start=datetime(2016, 1, 1, tzinfo=timezone.utc),
+                    State=ReservedInstanceState.active,
+                    UsagePrice=0.05,
+                    CurrencyCode=CurrencyCodeValues.USD,
+                    InstanceTenancy=Tenancy.default,
+                    OfferingClass=OfferingClassType.standard,
+                    OfferingType=OfferingTypeValues.Partial_Upfront,
+                    RecurringCharges=[
+                        RecurringCharge(Amount=0.05, Frequency=RecurringChargeFrequency.Hourly)
+                    ],
+                    Scope=scope.Availability_Zone,
+                )
+            ]
+        )
+
+    @handler("PurchaseReservedInstancesOffering", expand=False)
+    def purchase_reserved_instances_offering(
+        self,
+        context: RequestContext,
+        purchase_reserved_instances_offerings_request: PurchaseReservedInstancesOfferingRequest,
+    ) -> PurchaseReservedInstancesOfferingResult:
+        return PurchaseReservedInstancesOfferingResult(
+            ReservedInstancesId=long_uid(),
+        )
+
+    @handler("DeleteVpcEndpoints", expand=False)
+    def delete_vpc_endpoints(
+        self, context: RequestContext, delete_vpc_endpoints_request: DeleteVpcEndpointsRequest
+    ) -> DeleteVpcEndpointsResult:
+        return moto.call_moto(context)
```

Comment:
nit: Since we're using `MotoFallbackDispatcher`, the explicit `moto.call_moto(..)` should not be required here, right?

Reply:
Good catch. Fixed in [3da0fea](https://github.com/localstack/localstack/pull/5608/commits/3da0fea4bf9d496a998bd1c4217775d7a0d75f8c)